tests/ecmp/test_fgnhg.py

```diff
@@ -231,6 +258,32 @@ def fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank
 
     for dst_ip in dst_ip_list:
         partial_ptf_runner(ptfhost, 'initial_hash_check', dst_ip, exp_flow_count)
+  
+    if WARM_BOOT_TESTING == True:
+        warm_reboot_start_time = time.asctime( time.localtime(time.time()) ) 
+        #str(datetime.now().strftime('%Y-%m-%d-%H:%M:%S'))
```

Comment:
Is this a debug change that should be removed?

Reply:
Removed